app/Models/LdapAd.php

```diff
@@ -0,0 +1,431 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Models;
+
+use Exception;
+use Adldap\Adldap;
+use App\Traits\UserTrait;
+use Adldap\Query\Paginator;
+use Illuminate\Support\Collection;
+use Illuminate\Support\Facades\Log;
+use Adldap\Models\User as AdldapUser;
+use Adldap\Models\ModelNotFoundException;
+
+/**
+ * LDAP queries.
+ *
+ * @author Wes Hulette <jwhulette@gmail.com>
+ *
+ * @since 5.0.0
+ */
+class LdapAd extends LdapAdConfiguration
+{
+    use UserTrait;
+
+    /**
+     * @see https://wdmsb.wordpress.com/2014/12/03/descriptions-of-active-directory-useraccountcontrol-value/
+     */
+    const AD_USER_ACCOUNT_CONTROL_FLAGS = ['512', '544', '66048', '66080', '262656', '262688', '328192', '328224'];
```

Comment:
I don't know what these things do but I assume they're AD things and that makes me happy!!